confluent_kafka/src/Consumer.c

```diff
@@ -756,6 +815,11 @@ static PyObject *Consumer_close (Handle *self, PyObject *ignore) {
 
         rd_kafka_consumer_close(self->rk);
 
+        if (self->u.Consumer.rkqu) {
+                rd_kafka_queue_destroy(self->u.Consumer.rkqu);
```

Comment:
indent looks weird.

Reply:
Will fix. I noticed that there are a bunch of tabs in the code currently, and I gather from the majority of the code and the librdkafka style guide that spaces are preferred - I can clean those up as well.